atom/browser/api/atom_api_window.cc

```diff
@@ -424,6 +424,16 @@ void Window::ShowDefinitionForSelection() {
 }
 #endif
 
+#if defined(OS_MACOSX) || defined(OS_LINUX)
```

Comment:
We should also enable the API on Windows, the `Widget::SetVisibleOnAllWorkspaces` is defined on Windows so we don't have to make code platform-dependent here, even though it actually does nothing,